internal/statestore/redis.go

```diff
@@ -294,25 +286,34 @@ func (rb *redisBackend) IndexTicket(ctx context.Context, ticket *pb.Ticket) erro
 			continue
 		}
 
-		var d float64
-
 		switch v.Kind.(type) {
 		case *structpb.Value_NumberValue:
-			d = v.GetNumberValue()
+			attributesToValue[attribute] = v.GetNumberValue()
 		default:
-			// TODO: investigate if we need to throw out an exception if attribute is not a value
```

Comment:
Add a todo somewhere indicating that we currently dont support anything but numeric values and in future we might. That way, its clear why we actually have a generically supported proto.